src/main/java/seedu/address/model/company/Company.java

```diff
@@ -22,7 +22,8 @@ public class Company {
     private final Email email;
 
     // Data fields
-
+    private final Date startDate;
+    private final Date deadline;
```

Comment:
I think deadline could be changed to endDate for better readability and consistency